app/services/usps_in_person_proofing/proofer.rb

```diff
@@ -24,7 +24,8 @@ def request_facilities(location)
       end.body
 
       facilities = parse_facilities(response)
-      dedupe_facilities(facilities)
+      sorted_facilities = sort_by_ascending_distance(facilities)
```

Comment:
do we maybe want to remove the dupes before sorting - so we don't have to sort unnecessary locations?